docs-source/README.md

```diff
@@ -0,0 +1,7 @@
+# Oracle WebLogic Server Kubernetes Operator Documentation
+
+***
+For information on updating and contributing to the operator
+documentation please see the _Documentation_ section of the
```

Comment:
documentation please see -> documentation, see (added a comma and removed the "please" to align with related stem sentences)

Reply:
Done.